docs/index.md

```diff
@@ -92,7 +92,7 @@ provider "artifactory" {
 The following arguments are supported:
 
 * `url` - (Optional) URL of Artifactory. This can also be sourced from the `ARTIFACTORY_URL` environment variable.
-* `access_token` - (Optional) This can also be sourced from the `ARTIFACTORY_ACCESS_TOKEN` environment variable.
+* `access_token` - (Optional) This can also be sourced from `JFROG_ACCESS_TOKEN` or `ARTIFACTORY_ACCESS_TOKEN` environment variables.
 * `api_key` - (Optional) API key for api auth. Uses `X-JFrog-Art-Api` header.
```

Comment:
This actually needs to be removed. It's no longer supported @danielmkn

Reply:
Didn't we decided to keep `api_key` because we need it for some feature?

Reply:
yes, we decided to keep the `api_key`.